Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -289,6 +289,19 @@ public partial class Scheduler : IPipScheduler, IPipExecutionEnvironment, IFileC
         private int m_maxUnresponsivenessFactor = 0;
         private DateTime m_statusLastCollected = DateTime.MaxValue;
 
+        /// <summary>
+        /// Output file contents which affected by the source change of this build. 
+        /// </summary>
+        /// <remarks>
+        /// This list should be initialized with the source change list provided by GBR.
+        /// </remarks>
+        private readonly ConcurrentBigSet<AbsolutePath> m_sourceChangeAffectedOutputFiles = new ConcurrentBigSet<AbsolutePath>();

```

Comment:
Should `AbsolutePath` be `FileArtifact`?

Reply:
I don't see neccesarity of FileArtifact.